src/Discovery/DiscoveryListView.tsx

```diff
@@ -47,8 +47,9 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
       columns={props.columns}
       rowKey={props.config.minimalFieldMapping.uid}
       rowSelection={(
-        props.config.features.exportToWorkspace
-              && props.config.features.exportToWorkspace.enabled
+        (props.config.features.exportToWorkspace
+          && props.config.features.exportToWorkspace.enabled) || ((props.config.features.exportToWorkspace?.enableFillRequestForm
```

Comment:
we probably should consider renaming the config section `exportToWorkspace` to something else since it is more like an action button bar rather than just for exporting to ws
but this is more like a note to ourselves, not a item that in the scope of your PR

Reply:
Are you still planning to merge PR when all comments are resolved? We’d recommend that as we have a deadline to be met